### ISSUE TITLE
Native file picker

### ISSUE THREAD

**Issue by iandoug** [upscayl/upscayl#618]
### Checklist

- [X] I have checked that this issue isn't addressed in [the FAQ](https://github.com/upscayl/upscayl#-faq).
- [X] I have checked that this issue isn't addressed in any other issue or pull request.

### Description

Is there an option to use the native file picker (in my case, KDE) rather than the built-in one? Or is this a consequence of the underlying tech?

Thanks.

**Comment by aaronliu0130** [comment]
Upscayl should be using the native file picker unless you’re using flatpak/snap, which isolates the app from access to the file picker.

**Comment by iandoug** [comment]
So noted, thanks. I installed from Gentoo overlay edgets, will query with maintainer.

https://gpo.zugaina.org/media-video/upscayl/USE

**Comment by aaronliu0130** [comment]
Yes, that is indeed what it looks like. Could you send us a screenshot of Upscayl’s file picker and Gentoo’s file picker?

**Comment by iandoug** [comment]
Okular standard KDE picker

![okular](https://github.com/upscayl/upscayl/assets/12673886/0f38ce44-b2e3-47a8-8a37-1776f1c37cf3)

GIMP Gnome-on-KDE picker

![gimp](https://github.com/upscayl/upscayl/assets/12673886/714321a8-1cf5-4586-a2d3-12684548efad)

Upscayl

![upscayl](https://github.com/upscayl/upscayl/assets/12673886/6710c79d-7e1f-4489-85cb-4bbade0d5573)

The part that annoys me the most, and not really shown in the shots, is the sorting ... sorted alphabetically, so things starting with Caps come first, and folders mixed in with files. I'm used to folders first, and natural sorting (ie alphabetical, but not ASCII-style Caps first).

I don't know if this is thanks to an option missing during compile or not. Also the install did not set the icon in the app launcher menu, but that's probably a Gentoo install script issue.

**Comment by aaronliu0130** [comment]
Hmm, I see an X logo in the final shot. Is there a possibility that you did not configure stuff for the X window system while the rest of your system uses Wayland?

**Comment by iandoug** [comment]
Sorry I am not up to speed with the subtleties between X11 and Wayland ... 

I installed xlsclients which shows this (machine is 'fractal"):

fractal  firefox-esr
fractal  upscayl

Your question sounds like something I should ask the person who wrote the install script. I did try to find their contact details but failed, will try again.

Thanks.
